src/main.js

```diff
@@ -6,33 +6,41 @@ import highland from 'highland'
 import { SphereClient } from 'sphere-node-sdk'
 
 import CONS from './constants'
-import MapCustomFields from './mapCustomFields'
+import MapCustomFields from './map-custom-fields'
```

Comment:
Thought that using dashes in filenames is the most used one?
It makes it really clear we are talking about a file and it fits very well with the module naming convention that also uses dashes.